### PR TITLE
Document PostGIS extension in BigAnimal

### PR DESCRIPTION
**Pull request by josh-heyer**
## What Changed?

Continues on from #4889 which was merged early

### CODE REVIEW

**Comment by ebgitelman** [comment]
> This doesn't seem to be correct. PostGIS is enabled in the Additional Settings tab, not the DB Configuration tab. This may or may not be correct for some arbitrary UI change coming.

@amrita-suresh Can you let me know if DJ's comment is correct or if the doc is correct as revised? Thank you!

File: product_docs/docs/biganimal/release/using_cluster/extensions.mdx [hunk 1]
```diff
@@ -69,7 +70,7 @@ EDB extensions:
 
 ## Installing extensions
 
-Use the [`CREATE EXTENSION`](https://www.postgresql.org/docs/current/sql-createextension.html) command to install extensions, except for the EDB Postgres Tuner (pg_tuner) extension. You enable edb_pg_tuner on the **DB Configuration** tab of the Create or Edit Cluster page of the BigAnimal portal.
+Use the [`CREATE EXTENSION`](https://www.postgresql.org/docs/current/sql-createextension.html) command to install most extensions. You must enable certain extensions, including the EDB Postgres Tuner (pg_tuner) extension and PostGIS on the **DB Configuration** tab of the Create or Edit Cluster page of the BigAnimal portal.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/extensions.mdx:73]:
```suggestion
Use the [`CREATE EXTENSION`](https://www.postgresql.org/docs/current/sql-createextension.html) command to install most extensions. You must enable certain extensions, including the EDB Postgres Tuner (pg_tuner) extension and PostGIS. Enable pg_tuner on the **DB Configuration** tab of the Create or Edit Cluster page of the BigAnimal portal. Enable PostGIS on the **Additional Settings** tab.
```

@amrita-suresh @djw-m All good now?

Comment (djw-m) [reply product_docs/docs/biganimal/release/using_cluster/extensions.mdx:73]: There doesn't seem to be a simple Enable pg_tuner on DB Configuration:

Screenshot 2023-11-14 at 17 41 20

I suggest more guidance on that issue. Apart from that, it's fine.

**Comment by amrita-suresh** [comment]
@josh-heyer yes!

**Pull request merged**